direction.h

```diff
@@ -0,0 +1,6 @@
+#ifndef DIRECTION_H
+#define DIRECTION_H
+
+enum class Direction {UP, DOWN, LEFT, RIGHT};
```

Comment:
Make it per line for every direction. e g. {
Up,
...
}

Reply:
done